ansible/roles/jenkins-worker/tasks/main.yml

```diff
@@ -19,6 +19,9 @@
     state: present
     line: ::1 localhost.localdomain localhost
 
+- name: create NODE_TEST_DIR directory
+  file: path=/home/{{ server_user }}/tmp state=directory
```

Comment:
I would have thought it should be chowned to `server_user`:`server_group` (assuming those vars exist).